### ISSUE TITLE
something went wrong message

### ISSUE THREAD

**Issue by HassanOPFT** [microsoft/vscode#114443]
Issue Type: <b>Bug</b>

this message pop up when i run any code: 
Oops, something went wrong.  Please report this bug with the details below.
Line/issues/new
PS C:\Users\HassanMustafa\Desktop\Programming\Cpp> cd 
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 f a \ D e s k t o p \ P r o g r a m m i n g \ C p p \ " Space ; Space i f Space ( $ ? ) Space { Space g + + Space m a i n . c p p Space - o Space m a i n Space } Space ; Space i f Space ( $ ? ) Space { Space . \ m a i n Space } Enter
 c l e a r Enter
 c d Space " c : \ U s e r s \ H a s s a n M u s t a f a \ D e s k t o p \ P r o g r a m m i n g \ C p p \ " Space ; Space i f Space ( $ ? ) Space { Space g + + Space m a i n . c p p Space - o Space m a i n Space } Space ; Space i f Space ( $ ? ) Space { Space . \ m a i n Space } Enter

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.        
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)      
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
---------------------

VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:34:46.910Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.88GB (8.90GB free)|
|Process Argv|--crash-reporter-id 126d08c3-e8bc-4f6a-a567-07e5f13e3607|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
include-autocomplete|ajs|0.0.4
save-typing|akh|0.1.0
All-Autocomplete|Ati|0.0.23
code-gnu-global|aus|0.2.2
doxdocgen|csc|1.1.0
prettier-vscode|esb|5.8.0
code-runner|for|0.11.2
auto-run-command|gab|1.6.0
better-cpp-syntax|jef|1.15.5
auto-save|mcr|1.0.3
theme-monokai-pro-vscode|mon|1.1.18
cpptools|ms-|1.2.0-insiders
cpptools-extension-pack|ms-|1.0.0
sublime-keybindings|ms-|4.0.7
code-spell-checker|str|1.10.2
vscode-icons|vsc|11.1.0
commandlist|yam|1.0.3
material-theme|zhu|3.9.12

(6 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
openlogontheside:30221877
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30211401
wsl2prompt:30224612
vstry244:30244299
pythonvsdeb440:30242242
unusedpromptcf:30224611
folderexplorer:30224614
openfilemenucf:30224648
pythonvsded773:30236629
pythonvspyt859:30241729

```

</details>

**Comment by HassanOPFT** [comment]
> (Experimental duplicate detection)
> Thanks for submitting this issue. Please also check if it is already covered by an existing one, like:
> 
> * [message pop up all the time (#112673)](https://www.github.com/microsoft/vscode/issues/112673)
> * [bug trying to run babel (#111473)](https://www.github.com/microsoft/vscode/issues/111473)

I didn't find any solution anywhere

**Comment by HassanOPFT** [comment]
No i didn't find a solution, they just said extension issue, which
extension should i delete ?

**Comment by Tyriar** [comment]
@TylerLeonhardt this exception was fixed in PS preview: https://github.com/PowerShell/vscode-powershell/issues/702, is there something we're doing wrong in core to cause this to happen?

**Comment by HassanOPFT** [comment]
What do you mean?

**Comment by TylerLeonhardt** [comment]
@HassanOPFT, @Tyriar was asing me that question :) 

@Tyriar the user doesn't have the PowerShell extension installed so that issue is unrelated to what this user is seeing. The issue that you linked was scoped to the PowerShell extension's Integrated Console.

@HassanOPFT Some questions for you:

1. Has VS Code's terminal ever worked for you?
2. What code were you running when you saw this?
3. What version of PowerShell are you using? (Run `$PSVersionTable`. If this doesn't work, does the terminal say "powershell.exe"?)
4. Do you see this issue in PowerShell outside of VS Code?

Adding @daxian-dbw for any additional ideas

**Comment by daxian-dbw** [comment]
@HassanOPFT, the issue you ran into was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

**Comment by HassanOPFT** [comment]
The problem surfaced again ☹️
